ili/utils/ndes_pt.py

```diff
@@ -329,12 +329,24 @@ def net_constructor(x_batch, theta_batch, prior):
         if x_normalize:
             x_mean = x_batch.mean(dim=0).to(device)
             x_std = x_batch.std(dim=0).to(device)
+
+            # avoid division by zero
+            x_std[x_std == 0] = 1
+            x_std = torch.clamp(x_std, min=1e-16)
```

Comment:
Why do we have both `x_std[x_std == 0] = 1` and `x_std = torch.clamp(x_std, min=1e-16)`? If we just want to avoid dividing by small numbers, why not just the latter?

Reply:
This initially came from the thinking that, if the x is a constant, then we don't want to any rescaling (i.e. avoid dividing by 1e-16). But I now realize that the mean scaling will make all the x's=0, and so division by any x_std won't matter anyway. So, I've removed this first line in both instances.